charts/cp-zookeeper/templates/statefulset.yaml

```diff
@@ -65,16 +65,11 @@ spec:
       - name: {{ template "cp-zookeeper.name" . }}-server
         image: "{{ .Values.image }}:{{ .Values.imageTag }}"
         imagePullPolicy: "{{ .Values.imagePullPolicy }}"
-#        readinessProbe:
-#          exec:
-#            command: ['/bin/bash', '-c', 'echo "ruok" | nc -w 2 -q 2 localhost 2181 | grep imok']
-#          initialDelaySeconds: {{ .Values.probeInitialDelaySeconds }}
-#          timeoutSeconds: {{ .Values.probeTimeoutSeconds }}
-#        livenessProbe:
-#          exec:
-#            command: ['/bin/bash', '-c', 'echo "ruok" | nc -w 2 -q 2 localhost 2181 | grep imok']
-#          initialDelaySeconds: {{ .Values.probeInitialDelaySeconds }}
-#          timeoutSeconds: {{ .Values.probeTimeoutSeconds }}
+        livenessProbe:
```

Comment:
Should this probe have been removed also?

Reply:
Yes

Reply:
Thanks, Will i create a PR to remove?

Reply:
Definitely! Thanks a lot for the discovery and incoming PR :) Please also keep doc and values.yaml synced with the change

Reply:
https://github.com/confluentinc/cp-helm-charts/pull/159